src/drf_yasg/inspectors/field.py

```diff
@@ -72,15 +77,34 @@ def _has_ref_name(self, serializer):
 
     def field_to_swagger_object(self, field, swagger_object_type, use_references, **kwargs):
         SwaggerType, ChildSwaggerType = self._get_partial_types(field, swagger_object_type, use_references, **kwargs)
+        
+        description = getattr(field, 'description', getattr(field, '__doc__'))
+        
+        if description is None and hasattr(field, 'Meta') and hasattr(field.Meta, 'model') and hasattr(field.Meta.model, '__doc__'):
+            description = field.Meta.model.__doc__
```

Comment:
This doesn't seem like a good idea, because the Model is an internal entity and may contain help that may not be relevant to the API at all.